src/components/contestants.less

```diff
@@ -0,0 +1,274 @@
+@import "../main";
+
+
+.contestants {
+  .blue-section-wrapper {
+    .blue-section-wrapper (@img_path: "../../assets/img/contestants.png");
+  }
+
+  .blue-section {
+    .blue-section(@min-height: 500px);
+    color: @white;
+
+    h1 {
+      font-size: 6em;
+    }
+  }
+
+  .stats-section {
+    @section-height: 112px;
+    margin-top: -@section-height/2;
+
+    text-transform: uppercase;
+    color: @yellow;
+    text-align: center;
+
+    .inner-stats {
+      background-color: @light-black;
+
+      .description {
+        margin-top: 20px;
+        font-weight: bold;
+        color: @white;
+      }
+
+      .value {
+        font-weight: bold;
+        margin-top: -20px;
+        font-size: 3em;
+        width: 100%;
+        height: 100%;
+      }
+
+      .border-left:after {
+        // http://stackoverflow.com/questions/3953293/how-to-control-border-height
+        content: "";
+        border-left: 2px solid @grey;
+        position: absolute;
+        height: 50%;
+        left: 0;
+        top: 25%;
+      }
+    }
+  }
+
+  .white-section {
+    &.hidden {
```

Comment:
Exista deja clasa asta in bootstrap.
http://getbootstrap.com/css/#helper-classes-show-hide